tools/webpack.config.dev.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var version = require('./../package.json').version;
+var path = require('path');
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+
+var rootPath = path.join(__dirname, '..');
+
+function root(p){
+  return path.join(rootPath, p);
+}
+
+var distPath = root('dist');
+
+module.exports = {
+  entry: [
+    'webpack-hot-middleware/client?reload=true',
+    './src/entry'
+  ],
+  output: {
+    path: distPath,
+    publicPath: '/dist',
+    filename: 'bundle.js'
+  },
+  plugins: [
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoErrorsPlugin(),
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify('development'),
+      APP_VERSION: JSON.stringify(version)
+    })
+  ],
+  resolve : {
+    alias : {
+      'components' : root('src/components'),
+      'sass' : root('src/sass'),
+      'fonts': root('src/fonts')
+    }
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        loader: 'babel',
+        query: {
+          stage: 2,
+          optional: ['es7.classProperties'],
+          loose: 'all',
+          plugins: ['react-transform'],
+          extra: {
+            'react-transform': {
+              'transforms': [
+                {
+                  'transform': 'react-transform-hmr',
+                  'imports': ['react'],
+                  'locals': ['module']
+                }, {
+                  'transform': 'react-transform-catch-errors',
+                  'imports': ['react', 'redbox-react']
+                }
+              ]
+            }
+          }
+        }
+      },
+      {
+        test: /\.json$/,
+        loader: 'json'
+      },
+      {
+        test: /\.scss$/,
+        loader: 'style!css!autoprefixer?browsers=last 2 versions!sass'
```

Comment:
This thing manages the scss imports.
It transforms it to css, uses autoprefixer and then **includes it in the output page**.
It will be different for production mode